commcare_connect/opportunity/tasks.py

```diff
@@ -395,6 +398,16 @@ def bulk_update_visit_status_task(self, opportunity_id: int, file_path: str, fil
         messages = [f"Visit status updated successfully for {len(status)} visits."]
         if status.missing_visits:
             messages.append(status.get_missing_message())
+
+        if tracking_info:
+            tracking_info = GATrackingInfo.from_dict(tracking_info)
+            events = [
+                Event("visit_import_approved", {"updated": status.approved_count, "total": len(status.seen_visits)}),
+                Event("visit_import_rejected", {"updated": status.rejected_count, "total": len(status.seen_visits)}),
```

Comment:
Just curious, (it's probably discussed over while speccing) but to me these seems more like logging info than analytics, I wonder how we plan to use these in analytics.

Reply:
These metrics will be used to identify usage of "Exports" vs the "Bulk Approval" UI that we have built.